src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/LocalizableStrings.resx

```diff
@@ -168,4 +168,7 @@
   <data name="CmdVulnerableDescription" xml:space="preserve">
     <value>Lists packages that have known vulnerabilities. Cannot be combined with '--deprecated' or '--outdated' options.</value>
   </data>
+  <data name="CmdFormatDescription" xml:space="preserve">
+    <value>Specify output format type for list packages command. Requires the "console", "json" option.</value>
```

Comment:
```suggestion
    <value>Specifies the output format type for the list packages command.</value>
```

Minor grammar change to bring this in line with other descriptions for options and arguments for this command. I also removed the example formats - if you implement the enum as I mentioned above System.CommandLine will tell the user what the allowed values are.

Reply:
Sure, taken the suggestion.